src/splat/util/options.py

```diff
@@ -355,7 +357,15 @@ def _parse_yaml(
     p = OptParser(yaml)
 
     basename = p.parse_opt("basename", str)
-    platform = p.parse_opt_within("platform", str, ["n64", "psx", "ps2", "psp"])
+    unsupported_platform = p.parse_optional_opt("unsupported_platform", bool)
```

Comment:
Could you document this new option in https://github.com/ethteck/splat/blob/main/docs/Configuration.md ?